components/LeagueEntries/LeagueEntries.tsx

```diff
@@ -15,68 +16,100 @@ import Link from 'next/link';
- * @param props.teamLogo - the team logo
+ * @param props.userPickHistory - the user's pick history for this entry
+ * @param props.selectedTeamLogo - the team logo
  * @returns {React.JSX.Element} - A div element that contains the user's entry information
  */
 const LeagueEntries = ({
   entryName,
   linkUrl,
   isEliminated = false,
   isPickSet = false,
-  teamLogo = '',
-}: ILeagueEntriesProps): JSX.Element => (
-  <div
-    data-testid="league-entry-container-card"
-    className={cn(
-      'league-entry-container-card grid h-20 min-w-fit grid-cols-2 justify-between rounded-lg border border-border bg-card p-4 text-card-foreground shadow-sm',
-      isEliminated ? 'bg-muted' : 'transparent',
-    )}
-  >
-    <section
-      className="league-entry-header flex items-center gap-12"
-      data-testid="league-entry-header"
+  userPickHistory,
+  selectedTeamLogo = '',
+}: ILeagueEntriesProps): JSX.Element => {
+  return (
+    <div
+      data-testid="league-entry-container-card"
+      className={cn(
+        'league-entry-container-card grid h-21 min-w-fit items-center justify-between rounded-lg border border-border bg-card p-4 text-card-foreground shadow-sm',
+        isEliminated ? 'bg-muted' : 'transparent',
+        userPickHistory.length === 0 ? 'grid-cols-2' : 'grid-cols-3',
+      )}
     >
-      <h4
-        data-testid="league-entry-number"
-        className={cn(
-          'league-entry-number text-xl font-semibold leading-none tracking-tight',
-          isEliminated ? 'opacity-50' : '',
-        )}
+      <section
+        className="league-entry-header flex items-center gap-12"
+        data-testid="league-entry-header"
       >
-        {entryName}
-      </h4>
-      <EntryStatus isEliminated={isEliminated} />
-    </section>
-    <section
-      className="league-entry-footer flex items-center justify-end gap-12"
-      data-testid="league-entry-footer"
-    >
-      {isPickSet && (
-        <img
-          className="league-entry-logo h-12 w-12"
-          data-testid="league-entry-logo"
-          src={teamLogo}
-          alt="teamLogo"
-        />
+        <h4
+          data-testid="league-entry-number"
+          className={cn(
+            'league-entry-number text-xl font-semibold leading-none tracking-tight',
+            isEliminated ? 'opacity-50' : '',
+          )}
+        >
+          {entryName}
+        </h4>
+        <EntryStatus isEliminated={isEliminated} />
+      </section>
+      {userPickHistory.length > 0 && (
+        <section
+          className="flex gap-2 overflow-x-scroll justify-end mr-2"
+          data-testid="user-pick-history"
+        >
+          {userPickHistory?.map((logoURL, index) => (
+            <div
+              key={logoURL}
+              className="flex flex-col h-[66px] items-center justify-center border border-border px-4 py-1 rounded-lg gap-1"
```

Comment:
nit: I wonder if adding `opacity-80` to make this not stand out as much as the current week's pick might be better for design and UX?

Reply:
@ryandotfurrer @shashilo what do you think.  Here it is with opacity-80

![CleanShot 2024-09-12 at 16 29 18](https://github.com/user-attachments/assets/f80f1efc-f47d-42b2-b903-542268473b8c)

Reply:
✅ updated.